rsc/src/main/scala/rsc/parse/scala/Scanners.scala

```diff
@@ -166,4 +166,22 @@ trait Scanners {
       in.nextToken()
     }
   }
+
+  def optTrailingComma(): Unit = {
```

Comment:
Do we really need to use `restore(snapshot)`?

Reply:
If we don't use restore(snapshot) then the scanner is out of sync with the in object.